src/Core/Content/Media/Core/Application/RemoteThumbnailLoader.php

```diff
@@ -61,31 +65,34 @@ public function load(iterable $media): void
         $mediaThumbnailSizes = $this->getMediaThumbnailSizes();
         $baseUrl = $this->getBaseUrl();
 
+        /** @var MediaEntity|PartialEntity $mediaEntity */
```

Comment:
please do not add `@var` annotations. you should rather change the annotation of the method itself as I mentioned before. `@param iterable<MediaEntity|PartialEntity> $media

Reply:
done.

Reply:
have you forgot to push? :slightly_smiling_face: it is still there